extensions/standard-processors/tests/resource_apps/EchoParameters.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <iostream>
+#include <chrono>
+#include <thread>
+
+int main(int argc, char** argv) {
+  if (argc < 3) {
+    std::cerr << "Usage: ./EchoParameters <delay milliseconds> <text to write>" << std::endl;
+    return 1;
+  }
+
+  std::cout << argv[2] << std::endl;
```

Comment:
does the delay only apply to not-the-first "<text to write>"? e.g. `./EchoParameters 500 "hello" "banana"`, would not actually delay `"hello"`, only `"banana"`